packages/pds/src/actor-store/index.ts

```diff
@@ -83,44 +53,46 @@ export class ActorStore {
   }
 
   async keypair(did: string): Promise<Keypair> {
-    const got = await this.keyCache.fetch(did)
-    if (!got) {
-      throw new InvalidRequestError('Keypair not found', 'NotFound')
-    }
-    return got
+    const { keyLocation } = await this.getLocation(did)
+    const privKey = await fs.readFile(keyLocation)
+    return crypto.Secp256k1Keypair.import(privKey)
   }
 
-  async db(did: string): Promise<ActorDb> {
-    const got = await this.dbCache.fetch(did)
-    if (!got) {
+  async openDb(did: string): Promise<ActorDb> {
+    const { dbLocation } = await this.getLocation(did)
+    const exists = await fileExists(dbLocation)
+    if (!exists) {
```

Comment:
If sqlite throws an error when the db doesn't exist, perhaps we could use that rather than the eager check.  Would just save some ticks to open a db.

Reply:
I don't think it does unfortunately :/

And actually, calling `getDb` would cause the database to be _created_ i beliee